src/AppBundle/Entity/ClmAccount.php

```diff
@@ -50,14 +50,32 @@ class ClmAccount
      * User constructor.
      * @param string $name
      */
-    public function __construct($name)
+    public function __construct($name = '')
```

Comment:
The attribute ```accountName``` isn't part of id (and also shouldn't be) but it's not nullable so it's true that it have to be set to persist the object into database.

I would prefer to omit setting params in constructor as long as we do not use DDD approach or have another good reason to do so.